MachUpskillingFY17/JabbR-Core
# Justine/refactor persisted repo

Changes:
 - Deleted InMemoryRepository
 - Renamed PersistedRepository to Repository
 - Refactored all tests to use an in memory database with the repository
 - Ensured Travis build is passing

To Test:
 - All current functionality from dev should still be present
 - Run all tests and make sure they pass
